### PR TITLE
Feature/45/checkout shipping page

### PR DESCRIPTION
**Pull request by schleichardt**

### CODE REVIEW

**Comment by schleichardt** [comment]
@lauraluiz please review

get a product:
http://sphere-sunrise-dmexco-checkout.herokuapp.com/en/michael-kors-shopper-jetsettravel-30S3GTVT6L-red-A0E200000001YM5.html

add it to the cart.

go to the cart:
http://sphere-sunrise-dmexco-checkout.herokuapp.com/en/cart

click there checkout  or go to http://sphere-sunrise-dmexco-checkout.herokuapp.com/en/checkout/shipping

File: common/app/common/templates/PlayMessagesResolver.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+import com.github.jknack.handlebars.ValueResolver;
+import io.sphere.sdk.models.Base;
+import play.i18n.Messages;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+
+public class PlayMessagesResolver extends Base implements ValueResolver {
```
Comment (lauraluiz) [inline common/app/common/templates/PlayMessagesResolver.java]: So you are introducing this functionality via this PR?

Comment (schleichardt) [inline common/app/common/templates/PlayMessagesResolver.java]: :smiling_imp: Yes

**Comment by lauraluiz** [comment]
:+1:

**Pull request merged**